### PR TITLE
Add DS_Store to .gitignore

### PR DESCRIPTION
**Pull request by jhadvig**
Put an asterisk to ignore all the occurrences of the file in the repo tree.
/assign @spadgett

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,4 @@
 /frontend/npm-debug.log
 /frontend/yarn-error.log
 /frontend/gui_test_screenshots
+*.DS_Store
```
Comment (spadgett) [inline .gitignore]: can you move this near the top of the file before the directories?

Comment (jhadvig) [reply .gitignore]: can do 👍

**Comment by jhadvig** [comment]
@spadgett comment addressed

File: .gitignore [hunk 2]
```diff
@@ -2,6 +2,7 @@
 .idea
 .vscode
 .protractor-fail-fast
+*.DS_Store
```
Comment (spadgett) [inline .gitignore]: I think this should just be...

```suggestion
.DS_Store
```

(Sorry for missing that on my first review.)

**Comment by jhadvig** [comment]
@spadgett comment addressed

**Pull request merged**